README.md

```diff
@@ -32,7 +32,7 @@ docker pull rosukraine/leobot:latest
 ```
  7. Use the following command to start Docker container
 ```bash
-docker run -it --name leobot_dev -p 8080:8080 -p 8090:8090 -e DISPLAY -e LOCAL_USER_ID=$(id -u) -v /tmp/.X11-unix:/tmp/.X11-unix:rw rosukraine/leobot:latest
+docker run -it --name leobot_dev -p 8080:8080 -p 8090:8090 --device=/dev/input/js0 -e DISPLAY -e LOCAL_USER_ID=$(id -u) -v /tmp/.X11-unix:/tmp/.X11-unix:rw rosukraine/leobot:latest
```

Comment:
Please add here explanation that this only works in case if you have USB joystick connected :)